.golangci.yml

```diff
@@ -9,6 +9,7 @@ linters:
 
     # deprecated/replaced
     - deadcode
+    - exhaustruct
```

Comment:
I suppose this was added because of the `types.Info` struct added to `gosourcefile.go`. I'd prefer this to be an exeptional case to the linter and disable it inline, explaining the reason.

Reply:
Oh, I've added this one because `exhaustivestruct` had been added just in the line below this one. Now I'm thinking your intention was to add `exhaustivestruct` because it has been made obsolete by `exhaustruct`...?

Reply:
Yeah, that list there is only reflecting the obsolete/deprecated linters.